packages/jaspr_tailwind/lib/builder.dart

```diff
@@ -34,8 +34,6 @@ class TailwindBuilder implements Builder {
     var assets = await buildStep.findAssets(Glob('{lib,web}/**.dart')).toList();
     await Future.wait(assets.map((a) => buildStep.canRead(a)));
```

Comment:
When using a custom config with other content files, the tailwind command won't be rerun when those other files change. Do you think this is a problem?

Reply:
Well, it may be a bit cumbersome to use, but since the main files are dart files, I don't think it's a serious problem. If you were to modify this part, you would have to read the config js file and extract the content part in dart. Is it possible to implement this?

Reply:
I think its ok for now, but a welcome improvement for a future PR if you want. So I'm going to merge this now and also add a hint that this isn't watching any additional files added to content.